lib/atlas/dataset/derived_dataset.rb

```diff
@@ -15,11 +15,16 @@ def graph
       @graph ||= GraphDeserializer.build(YAML.load_file(graph_path))
     end
 
-    def initializer_inputs
-      InitializerInput
-        .collection_from_hash(init)
-        .sort_by(&:priority)
-        .reverse
+    def init
+      Hash[super.sort_by do |initializer_input, _|
+        initializer_input.priority * -1
```

Comment:
[ETE sorts normal inputs](https://github.com/quintel/etengine/blob/470256764853fa338a13543e272de324bd46610b/app/models/scenario/user_updates.rb#L96); from Atlas' point of view `priority` is just another attribute with no special meaning. Only when ETE executes an input does the priority have any significance.

Other finder methods (e.g. `all`) don't provide any guarantees about the order of returned objects.

Is there a reason for `DerivedDataset#init` to be different? Does Atlas need the inputs to be in priority order? If we do one day add the ability to customize initializer inputs through ETModel, ETE will have to do the sorting anyway.